### DISCUSSION TITLE
How to obtain milvusdb/milvus:v2.3.17  images?

### DISCUSSION THREAD

**Discussion by khq007110** [milvus-io/milvus#33734] [Q&A and General discussion] [answered]
I am unable to pull the Milvus image using the command docker pull milvusdb/milvus:v2.3.17 in China. Are there any other ways to obtain the image?

**Comment by xiaofan-luan** [comment, marked as answer]
we upgrade all the image to official docker hub https://hub.docker.com/r/milvusdb/milvus/tags?page=&page_size=&ordering=&name=2.3.17

**Reply by niurunze** [reply]
I can't access this address, how should I get it?

**Reply by JackLCL** [reply]
> I can't access this address, how should I get it?

@niurunze  Please send an email to community@zilliz.com, you will get the latest milvus docker image.
![image](https://github.com/milvus-io/milvus/assets/53512883/88cd539e-b85b-4134-a347-57eb4f715e5f)
